ProductDock/rbc3-library-fe
# RBN-91: Add write a review form

### What did you implement?

Implemented write a review form, with rate section, recommend book checkboxes and review input field.

### How can we verify the changes?

Changes can be verified locally by adding a button to the testing page.  
In the button’s click event handler, invoke the ReviewForm component. 
Ensure that clicking the button opens the drawer and displays the ReviewForm component as expected.

### More info

- Jira: [RBN-91](https://productdock.atlassian.net/jira/software/projects/RBN/boards/11?selectedIssue=RBN-91) 

- Figma: [Design-web view](https://www.figma.com/design/x4SgW80BE24s9gAFHM3gXS/RBC---Novi-Sad%2C-2024?node-id=2339-2143&t=EWVrVPX6dhETw5Aj-4)
               [Design- mobile view](https://www.figma.com/design/x4SgW80BE24s9gAFHM3gXS/RBC---Novi-Sad%2C-2024?node-id=3531-6862&t=EWVrVPX6dhETw5Aj-4)
